pipeline/brew/brew.go

```diff
@@ -85,7 +85,7 @@ func (Pipe) Description() string {
 
 // Run the pipe
 func (Pipe) Run(ctx *context.Context) error {
-	if ctx.Config.Brew.Repo == "" {
+	if ctx.Config.Brew.Repo == "" || ctx.Config.Brew.Repo == "none" {
```

Comment:
For brew you can, but GitHub is a requirement for brew, and I wanted to keep pairity (ie, if github = none, brew = none). Can't say there's a better way